src/services/lineService.ts

```diff
@@ -38,6 +44,22 @@ export default class LineService {
             return null;
         }
     }
+
+    /**
+     * Converts Apollo's queryResult into:
+     * @return {IExternalLine} externalLine
+     * @return {IExternalRoute[]} externalLine.externalRoutes
+     */
+    private static getExternalLine(line: any) {
```

Comment:
Would this be difficult to type?

Reply:
Yes it can't be typed because line !== externalRoutePathLine

(that line has field reittisByLintunnus and externalRoutePathLine doesn't)